src/kontrol/__main__.py

```diff
@@ -781,6 +783,13 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help='Reinitialize CFGs even if they already exist.',
     )
+    prove_args.add_argument(
+        '--skip-setup-reinit',
+        dest='skip_setup_reinit',
+        default=False,
+        action='store_true',
+        help='Do not reinitialize test setup; reuse most recent setup proof instead',
```

Comment:
This reads like "the most recent version of `setUp` proof that was ever generated", but, if I understood correctly, it's test-specific?

Reply:
It might also be worth mentioning that it should be used together with `--reinit`.